lib/rules/indentation/html.test.js

```diff
@@ -355,18 +384,20 @@ a {
 			],
 		},
 		{
-			code: `
-\t<style>
-\ta {
-\t\tdisplay:block;
-\t}
-\t</style>`,
-			fixed: `
-\t<style>
-\t\ta {
-\t\t\tdisplay:block;
-\t\t}
-\t</style>`,
+			code: stripIndent(`
```

Comment:
The same thing here - `stripIndent` cuts the leading indentation, making the test not equivalent to the previous one